docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -100,108 +101,49 @@ Make sure that you have appropriate tagging privileges that control your ability
-   k8s-master   Ready    master   32m   v1.14.2   zone-a   region-1
-   k8s-node1    Ready    <none>   18m   v1.14.2   zone-a   region-1
-   k8s-node2    Ready    <none>   18m   v1.14.2   zone-b   region-1
-   k8s-node3    Ready    <none>   18m   v1.14.2   zone-b   region-1
-   k8s-node4    Ready    <none>   18m   v1.14.2   zone-c   region-1
-   k8s-node5    Ready    <none>   18m   v1.14.2   zone-c   region-1
-   ```
-
-4. Install the CSI driver.
-
-   Make sure `external-provisioner` is deployed with the arguments `--feature-gates=Topology=true` and `--strict-topology`.
-
-   In the credential secret file, add entries for region and zone.
+1. In the [vsphere config secret](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/installation.html#create_csi_vsphereconf) file, add entries for region and zone.
 
    ```bash
    [Labels]
    region = k8s-region
    zone = k8s-zone
    ```
 
-   Make sure secret is mounted on all workload nodes as well. This is required to help node discover its topology.
-
-   ```yaml
-     containers:
-     - name: vsphere-csi-node
-     .
-     .
-       env:
-       - name: VSPHERE_CSI_CONFIG
-         value: "/etc/cloud/csi-vsphere.conf"
-     .
-     .
-       volumeMounts:
-       - name: vsphere-config-volume
-         mountPath: /etc/cloud
-         readOnly: true
-     .
-     .
-     volumes:
-     - name: vsphere-config-volume
-       secret:
-         secretName: vsphere-config-secret
-     .
-     .
-     ```
-
-5. Verify that your CSI driver installation is successful.
-
-   ```bash
-   kubectl get csinodes -o jsonpath='{range .items[*]}{.metadata.name} {.spec}{"\n"}{end}'
-   k8s-node1 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node1 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
-   k8s-node2 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node2 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
-   k8s-node3 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node3 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
-   k8s-node4 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node4 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
-   k8s-node5 map[drivers:[map[name:csi.vsphere.vmware.com nodeID:k8s-node5 topologyKeys:[failure-domain.beta.kubernetes.io/region failure-domain.beta.kubernetes.io/zone]]]]
-   ```
+2. Make sure `external-provisioner` is deployed with the arguments `--feature-gates=Topology=true` and `--strict-topology`.
+   - Uncomment lines in the yaml file marked with `needed only for topology aware setup`. - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.2.0/manifests/v2.2.0/deploy/vsphere-csi-controller-deployment.yaml#L160-L161
+
+3. Make sure secret is mounted on all workload nodes as well. This is required to help node discover its topology.
```

Comment:
The description for points 3 and 4 match.
Could we instead say - `Edit the CSI node daemonset YAMLs to support topology feature`

Reply:
updated point 4 to say 

> After the installation, verify `csinodes` objects has `topologyKeys`.